appserver/web/war-util/src/main/java/org/glassfish/web/loader/ServletContainerInitializerUtil.java

```diff
@@ -562,6 +566,17 @@ private static void scanDirectory(File dir, ClassDependencyBuilder classInfo) {
         return initializerList;
     }
 
+    private static Map<Class<? extends ServletContainerInitializer>, Set<Class<?>>> checkAgainstBlacklist(
+            Map<Class<? extends ServletContainerInitializer>, Set<Class<?>>> initializersList) {
+        Iterable<ServletContainerInitializerBlacklist> blacklistServices = ServiceLoader.load(
```

Comment:
Minor nitpick: variable should be "blacklistedServices"

Reply:
I nitpick back on that - that sounds like a list of services that are blacklisted, rather than a list of blacklist services.